config/routes/bops.rb

```diff
@@ -29,6 +29,16 @@
 
   mount BopsAdmin::Engine, at: "/admin", as: :bops_admin
 
+  scope path: "/admin", module: "bops_admin", as: "bops_admin" do
+    resource :notify, only: [:show, :update], controller: "notify"
+
+    get "notify/send_test", to: "notify#send_test_new", as: :notify_send_test
+    post "notify/send_test", to: "notify#send_test_create"
+
+    get "notify/preview_letter", to: "notify#preview_letter", as: :notify_preview_letter
+    post "notify/preview_letter", to: "notify#generate_preview"
+  end
+
```

Comment:
I don't understand rails routing very well, but can this go in `engines/bops_admin/config/routes.rb` instead?

Reply:
> I don't understand rails routing very well, but can this go in `engines/bops_admin/config/routes.rb` instead?

You're right.. looking at `engines/bops_admin/config/routes.rb`, I can see there is already a `notify` resource in there. I'll move it